package.json

```diff
@@ -10,6 +10,8 @@
   },
   "scripts": {
     "test": "standard && node--test",
+    "preversion": "./scripts/pre-version.js",
+    "postpublish": "./scripts/post-publish.js",
```

Comment:
Hmm, this is a bit fragile. Do we have any documentation on the release process? IIUC, you are assuming we will always be using `npm version` followed by `npm publish`, right? 

It would be great to describe this aspect in the docs for the release process. If we ever decide to change it, e.g. automate it via GitHub Actions, then the docs should remind us about this aspect.

FWIW, pnpm has the concept of [`publishConfig`](https://pnpm.io/package_json#publishconfig). Unfortunately, it does not allow overwriting custom fields like `sentryEnvironment`.

Reply:
We have documentation for it: https://github.com/filecoin-station/core#development If you `npm run release`, all of these steps are taken care of. But yes, this is assuming the release process runs `npm version` and then `npm publish`